.github/workflows/version.yml

```diff
@@ -37,90 +38,80 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       files: ${{ steps.changed_files.outputs.files }}
```

Comment:
`changed_files` is no longer a step id, this was changed to `generate_matrix` that return a `result` instead of `files`.

Can the build job be adjusted to use `steps.generate_matrix.outputs.result` instead?

Reply:
> Can the build job be adjusted to use `steps.generate_matrix.outputs.result` instead?

Well, not really since that one is now filtered by the files to be signed :cry: 

Also, changed files now is using json format.

I am actually wonder how the current code works, as the files are passed as json to the Cloudflare API. The output of `jitterbit/get-changed-files` is space delimited though (see https://github.com/jitterbit/get-changed-files). :thinking: 
```
--data '{"files": ${{ steps.flush.outputs.files }}}'
 ```

Reply:
It works because we create a JSON array here <https://github.com/home-assistant/version/blob/master/.github/workflows/version.yml#L162-L171>

Reply:
Ok, I've pushed a change which creates a file list similar to before. I guess there is another opportunity to build the CloudFlare purge list also as part of the prepare job. :thinking:

Reply:
Nice, I did not know you could use `core` in the script action 👍 
Yes, it can be done better for the URLs passed down there, but as this PR has already changed a lot, let's leave that to another time.